Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -2394,79 +2386,74 @@ private async Task UpdateTimersForSeriesTimer(SeriesTimerInfo seriesTimer, bool
             var allTimers = GetTimersForSeries(seriesTimer)
                 .ToList();
 
-            var registration = await _liveTvManager.GetRegistrationInfo("seriesrecordings").ConfigureAwait(false);
 
             var enabledTimersForSeries = new List<TimerInfo>();
-
-            if (registration.IsValid)
+            foreach (var timer in allTimers)
             {
-                foreach (var timer in allTimers)
+                var existingTimer = _timerProvider.GetTimer(timer.Id);
+
+                if (existingTimer == null)
                 {
-                    var existingTimer = _timerProvider.GetTimer(timer.Id);
+                    existingTimer = string.IsNullOrWhiteSpace(timer.ProgramId)
+                        ? null
+                        : _timerProvider.GetTimerByProgramId(timer.ProgramId);
+                }
 
-                    if (existingTimer == null)
+                if (existingTimer == null)
+                {
+                    if (ShouldCancelTimerForSeriesTimer(seriesTimer, timer))
+                    {
+                        timer.Status = RecordingStatus.Cancelled;
+                    }
+                    else
                     {
-                        existingTimer = string.IsNullOrWhiteSpace(timer.ProgramId)
-                            ? null
-                            : _timerProvider.GetTimerByProgramId(timer.ProgramId);
+                        enabledTimersForSeries.Add(timer);
                     }
+                    _timerProvider.Add(timer);
 
-                    if (existingTimer == null)
+                    if (TimerCreated != null)
                     {
+                        TimerCreated(this, new GenericEventArgs<TimerInfo>(timer));
+                    }
+                }
+                else
+                {
+                    // Only update if not currently active - test both new timer and existing in case Id's are different
+                    // Id's could be different if the timer was created manually prior to series timer creation
+                    ActiveRecordingInfo activeRecordingInfo;
```

Comment:
Is this actually used anywhere?

Reply:
I dunno. I have no way to test LiveTv I think, so I don't want to make too many changes in there. It seems it's only used to check the existence of some Id's.

Reply:
I mean the variable, if it's unused replace its name with an underscore